examples/differential_rotation/comparing_rotation_models.py

```diff
@@ -0,0 +1,81 @@
+"""
+======================================
+Comparing differential-rotation models
+======================================
+
+How to compare differential-rotation models.
+
+The example uses the `~sunpy.coordinates.metaframes.RotatedSunFrame` coordinate
+metaframe in `sunpy.coordinates` to apply differential rotation to a coordinate.
+See :ref:`sunpy-coordinates-rotatedsunframe` for more details on using
+`~sunpy.coordinates.metaframes.RotatedSunFrame`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import HeliographicStonyhurst, RotatedSunFrame
+from sunpy.data.sample import AIA_335_IMAGE
+from sunpy.sun.constants import sidereal_rotation_rate
+
+##############################################################################
+# First, we use an AIA observation primarily as a pretty background.  We also
+# define the meridian using a two-element coordinate array of the south pole
+# and the north pole at zero longitude.
+
+aiamap = sunpy.map.Map(AIA_335_IMAGE)
+meridian = SkyCoord(0*u.deg, [-90, 90]*u.deg, frame=HeliographicStonyhurst,
+                    obstime=aiamap.date)
+
+##############################################################################
+# Next, we calculate the sidereal rotation period of the Sun.  This is the
+# time for a full rotation relative to an inertial reference frame (e.g.,
+# distant stars), as opposed to the synodic period, which is the apparent
+# rotation period as seen from an Earth-based observer.  Since the Earth
+# orbits the Sun in the same direction as the Sun rotates, the Sun appears to
+# rotate slower for an Earth-based observer.
+
+sidereal_period = 360*u.deg / sidereal_rotation_rate
+print(sidereal_period)
+
+##############################################################################
+# We use `~sunpy.coordinates.metaframes.RotatedSunFrame` to rotate the
+# meridian by one sidereal period using each of the available
+# differential-rotation models.  See
+# :func:`~sunpy.physics.differential_rotation.diff_rot` for details on each
+# model.
+
+rotated_meridian = {}
+for model in ['howard', 'snodgrass', 'allen', 'rigid']:
+    rotated_meridian[model] = SkyCoord(RotatedSunFrame(base=meridian,
+                                                       duration=sidereal_period,
+                                                       rotation_model=model))
+
+##############################################################################
+# Plot all four differential-rotated meridians over the original plot.  To
```

Comment:
I think this sentence could be removed. It reads a bit awkward and the next one pretty much says the same.

Reply:
Okay, I've polished the text a bit.